### ISSUE TITLE
color contrast is not accessible

### ISSUE THREAD

**Issue by evanwill** [LibraryCarpentry/lc-second-website#4]
The contrast of text versus the background is an important usability consideration as it makes it difficult or impossible for people to read. If the contrast is too low it will fail web accessibility standards. There are are few places where the current color combinations will fail accessibility tests for color contrast. 

1. The most critical is the use of the yellow color (#FED136) for body links over a white background (here is the failing [contrast check](https://webaim.org/resources/contrastchecker/?fcolor=FED136&bcolor=FFFFFF). I think a different color choice for body links with more contrast is important.

2. The reverse combination used on "buttons" in the navigation will also fail the tests, but appears more readable because of the larger text size. One simple higher contrast option would be to have the active navigation item to use black text (#000000) with the theme yellow (#FED136) ([passing test](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FED136)).

3. The other low contrast area I noted was the dark gray LC logo (#7B7979) in the upper left with a light black background (#222) in the nav bar ([failing test](https://webaim.org/resources/contrastchecker/?fcolor=7B7979&bcolor=222222)). A higher contrast option would be to push the LC logo closer to white.

**Comment by weaverbel** [comment]
I have changed the colour from yellow to grey/blue and the contrast is much better. @evanwill @richyvk What do you think?

**Comment by richyvk** [comment]
I hate to do this but I am actually colourblind and that blue is much less obvious in links to me than the yellow. I would go so far as to say it is not very clear what is a link :(

There must be a colour that is both nice and completely visible to genetic defects like me!

**Comment by weaverbel** [comment]
Do you have a suggestion for a colour that would work @richyvk ?

**Comment by weaverbel** [comment]
I have tried red now @richyvk @jezcope @evanwill

**Comment by evanwill** [comment]
The red contrast passes the accessibility tests. However, the blue is still on :hover (?) and messing things up if you hover or click, then scroll around.

Do we have an official color scheme? 

The general "rule-of-thumb" color-blind accessibility guideline is avoid these combos:
- Green & Red
- Green & Brown
- Blue & Purple
- Green & Blue
- Light Green & Yellow
- Blue & Grey
- Green & Grey
- Green & Black
